generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.state.ts

```diff
@@ -137,6 +143,15 @@ export const <%= entityInstance %>EditState = {
         let <%= entityInstance %>Service: <%= entityClass %>Service = trans.injector().get(<%= entityClass %>Service);
         let id = trans.params()['id'];
         <%= entityInstance %>Service.find(id).subscribe(<%= entityInstance %> => {
+            <%_ if(hasDate) { _%>
+            <% for (idx in fields) { if (fields[idx].fieldType == 'LocalDate') { %>
+            <%= entityInstance %>.<%=fields[idx].fieldName%> = {
```

Comment:
I can do this, but I don't know if it's a good idea ! Anybody can use the find method to get an entityInstance but it's only in the modal that we want to format the date so that it works with NgbDatePicker.

Ps : NgbDatepIcker only accepts objects with this format  { year: xxx, month: xxx, day: xxx}, and what we receive from find is a Date object (I hope I was clear enough)